src/common/components/composite/layout/TopBar/SearchBar.tsx

```diff
@@ -31,7 +31,13 @@ const SearchBar = () => {
     setValue('');
   }, [match, query]);
 
+  const isTermEmpty = useMemo(() => value.length < MINIMUM_TERM_LENGTH, [value]);
```

Comment:
Should be named isTermValid or isTermLengthValid.

Reply:
Agreed. Got carried away.